tests/integration/test_charm.py

```diff
@@ -395,7 +399,7 @@ async def test_log_rotate(ops_test: OpsTest) -> None:
         universal_newlines=True,
     )
 
-    log_rotated = "audit.log.1.gz" in log_files
+    log_rotated = "audit.log.1" in log_files
```

Comment:
QQ why was this needed?

Reply:
The change we discussed about the `delaycompress` option in logrotate.
Without `delaycompress` we have:
```
audit.log audit.log.1.gz audit.log.2.gz
```
With  `delaycompress` we have:
```
audit.log audit.log.1 audit.log.2.gz
```